### PR TITLE
Updating scikit-image Community Calls

### PR DESCRIPTION
**Pull request by alexdesiqueira**
Updating scikit-image Community Calls according to the results in [crab.fit](https://crab.fit/scikitimage-community-call-251901).

### CODE REVIEW

File: calendars/skimage.yaml [hunk 1]
```diff
@@ -1,16 +1,27 @@
 name: scikit-image Community Calendar
 timezone: America/Los_Angeles
 events:
+  - summary: scikit-image Community Call (Europe/Africa)
+    description: |
+      The scikit-image Community Call. Friendly for Europe/Africa.
+
+      Meeting notes: https://hackmd.io/@alexdesiqueira/HyyHcIjQ9
```
Comment (tupui) [inline calendars/skimage.yaml:8]: FYI, if wanted in HackMD you can actually rename the link. Click on the HyyHc... and you can put something more readable. As you wish OC 😃

Comment (alexdesiqueira) [reply calendars/skimage.yaml:8]: :thinking: yeah, I remember that existed, Pamphile! I didn't find the option though :laughing:

Comment (tupui) [reply calendars/skimage.yaml:8]: If you're admin on the page you should see this. In the example I have typed in the box `scipy-meetup`
Screenshot 2022-06-07 at 00 09 38

Comment (alexdesiqueira) [reply calendars/skimage.yaml]: @tupui aha! Of course, it was hidden in the share options :laughing: Thank you!

**Review by tupui — APPROVED** [review]
Perfect, thanks @alexdesiqueira

File: calendars/skimage.yaml [hunk 2]
```diff
@@ -1,16 +1,27 @@
 name: scikit-image Community Calendar
 timezone: America/Los_Angeles
 events:
+  - summary: scikit-image Community Call (Europe/Africa)
+    description: |
+      The scikit-image Community Call. Friendly for Europe/Africa.
```
Comment (stefanv) [inline calendars/skimage.yaml:6]:
```suggestion
      The scikit-image Community Call. Timezone-friendly for Europe/Africa.
```

File: calendars/skimage.yaml [hunk 3]
```diff
@@ -1,16 +1,27 @@
...
+    begin: 2022-06-14 10:00:00
+    end: 2022-06-14 11:00:00
+    url: https://berkeley.zoom.us/j/4760246500?pwd=dC9jVVB0ZzlHT0VpdlBkc21oYmVUQT09
+    repeat:
+      interval:
+        days: 14
+      until: 2025-01-01 00:00:00
   - summary: scikit-image Community Call (Americas/Oceania)
     description: |
       The scikit-image Community Call. Friendly for Americas/Oceania.
```
Comment (stefanv) [inline calendars/skimage.yaml:18]:
```suggestion
      The scikit-image Community Call. Timezone-friendly for Americas/Oceania.
```

**Pull request merged**